frontend/talentsearch/src/js/components/support/SupportForm.tsx

```diff
@@ -235,11 +238,14 @@ const SupportFormApi = () => {
       toast.error(
         intl.formatMessage(
           {
-            defaultMessage: `Error: creating ticket failed (code {errorCode})`,
-            id: "C3Lv2t",
+            defaultMessage: `Sorry, something went wrong. Please email {emailAddress} and mention this error code: {errorCode}.`,
+            id: "tZLItl",
             description: "Support form toast message error",
           },
-          { errorCode: response.status },
+          {
+            emailAddress: TALENTSEARCH_SUPPORT_EMAIL,
+            errorCode: response.status,
+          },
```

Comment:
Can we add a `mailto:` link to the email address and extend how long the toast appears for or force it to be dismissed?

I had a hard time remembering the email address in the short amount of time it appeared for and trying to copy the email just dismissed the toast for me.

Reply:
Yep, the [toast](https://fkhadra.github.io/react-toastify/api/toast) api has options including `autoClose` ("Delay in ms to close the toast. If set to false, the notification needs to be closed manually"), how long did you have in mind to dely the close? I don't think we currently have anywhere else on the site where we don't use the parameters set in [ToastContainer](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/frontend/common/src/components/Toast/Toast.tsx#L32).

Reply:
Rule of thumb I was taught was enough time to read the message 3 times over. Took me about 7 seconds to read so 20 seconds should be enough. We can try it out and see if that is too long or not.

Reply:
See latest changes:

- [Add anchorTag to support form toast message error](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4450/commits/1ae68c83cc6664fade3052b16fd3042e7d1facfb)
- [Delay toast error autoClose for 20 seconds](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4450/commits/7ba5b71eba76fa69ddf416a73af060ead8ade17d)